addons/urdfreader/urdfreader.cc

```diff
@@ -18,7 +19,7 @@
   typedef urdf::JointSharedPtr JointPtr;
   typedef urdf::ModelInterfaceSharedPtr ModelPtr;
   typedef urdf::Joint UrdfJointType;
-
+    
```

Comment:
Superfluous whitespace that clutters the diff, should be removed before merging.